.github/ISSUE_TEMPLATE/2_refactor.yaml

```diff
@@ -33,7 +33,7 @@ body:
 
         - [**Metrics**](https://github.com/Lightning-AI/metrics):
           Machine learning metrics for distributed, scalable PyTorch applications.
-        - [**Lite**](https://pytorch-lightning.readthedocs.io/en/latest/starter/lightning_lite.html):
+        - [**Lite**](https://lightning.ai/docs/pytorch/latest/starter/lightning_lite.html):
```

Comment:
I get `404`, but is it going to be available after some other PR lands?

Reply:
```suggestion
```

Reply:
There are more uses of this link in this PR